src/DynamoPackages/Interfaces/IFileSystem.cs

```diff
@@ -7,8 +8,12 @@ namespace Dynamo.PackageManager
     /// </summary>
     public interface IFileSystem
     {
+        IEnumerable<string> EnumerateAllFiles(string dir);
+        IEnumerable<string> EnumerateAllDirectories(string dir);
```

Comment:
Should these be aligned better with the underlying system APIs and be called `GetFiles` and `GetDirectories`?